CONTRIBUTING.md

```diff
@@ -11,6 +11,20 @@ Please read [the full text](/CODE_OF_CONDUCT.md) so that you can understand what
 
 There are [many ways](https://material-ui.com/getting-started/faq/#material-ui-is-awesome-how-can-i-support-the-project) to contribute to Material-UI, code contribution is one aspect of it. For instance, documentation improvements are as important as code changes.
 
+## Trying changes on the documentation site
+
+The documentation site is built with Material-UI and contains examples of all the components.
+This is a great place to experiment with your changes.
+
+To get started:
+
+```sh
+yarn start
+```
+
+You can now access the documentation site [locally](http://localhost:3000).
+Changes to the docs will hot reload the site.
+
```

Comment:
One small problem - this now precedes the instructions for forking and cloning the repo